detection_rules/etc/packages.yml

```diff
@@ -1,44 +1,27 @@
 ---
 package:
-  name: "8.7"
```

Comment:
The `bump-pkg-versions` CLI command was used to update this. Unfortunately comments in YAML are not saved objects that can be exported and were automatically removed as a result.

Moving forward, we are looking to create a `prepare-for-next-release` workflow that does this PR for us and as a result, rely on programmatic YAML loading, editing and saving.